apps/prairielearn/src/lib/externalGraderResults.ts

```diff
@@ -129,9 +128,14 @@ async function loadQueueUrl(sqs) {
   return queueUrl;
 }
 
-async function processMessage(data) {
-  /** @type {string} */
-  let jobId;
+async function processMessage(data: {
+  jobId: string;
+  event: string;
+  data: {
+    receivedTime: string;
+  };
```

Comment:
I'm 50/50 on this. To do this "right" I'd rather declare a schema for the data coming in from the queue, parse data with that, then derive this type from that. As it stands, `processMessage` is really called with something of type `any` so this type gives us false confidence. I won't block on this but I lean slightly towards typing this as `any`.